lte/gateway/python/integ_tests/Makefile

```
@@ -54,6 +54,16 @@ else
 	$(foreach test,$(EXTENDED_TESTS) $(PRECOMMIT_TESTS),$(call execute_test,$(test));)
 endif
 
+.PHONY: fed_integ_test
+fed_integ_test: $(PYTHON_BUILD)/setupinteg_env $(BIN)/nosetests
```

There was a switch from `nose` to `pytest` for testing.

```suggestion
fed_integ_test: $(PYTHON_BUILD)/setupinteg_env $(BIN)/pytest
```